### PR TITLE
update guide to make the command works as expected

### LINKED ISSUE #80422: doc for "fake-gitserver" need to be updated
**What happened**:
Follow the "README", I hit below issue, the command there need a prefix "http" or "https".

```bash
# git clone localhost:8000
Cloning into 'localhost'...
root@localhost's password:
fatal: '30000' does not appear to be a git repository
fatal: Could not read from remote repository.
```

**What you expected to happen**:
clone a fake repo successfully.

**How to reproduce it (as minimally and precisely as possible)**:
- follow the "README" here.
https://github.com/kubernetes/kubernetes/tree/master/test/images/agnhost#fake-gitserver

- create a pod
- create a service
- git clone localhost:8000

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): latest
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):  all
- Kernel (e.g. `uname -a`): all

### PR DESCRIPTION
**Pull request by chendave**
Signed-off-by: Dave Chen <dave.chen@arm.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
user can run commands successfully by following the guide. 
**Which issue(s) this PR fixes**:
Fixes #80422

```release-note
NONE
```

### CODE REVIEW

File: test/images/agnhost/fakegitserver/gitserver.go [hunk 1]
```diff
@@ -27,8 +27,8 @@ import (
 var CmdFakeGitServer = &cobra.Command{
 	Use:   "fake-gitserver",
 	Short: "Fakes a git server",
-	Long: `When doing "git clone localhost:8000", you will clone an empty git repo named "8000" on local.
-You can also use "git clone localhost:8000 my-repo-name" to rename that repo.`,
+	Long: `When doing "git clone http://localhost:8000", you will clone an empty git repo named "8000" on local.
```
Comment (spiffxp) [inline test/images/agnhost/fakegitserver/gitserver.go]:
```suggestion
	Long: `When doing "git clone http://localhost:8000", you will clone an empty git repo named "localhost" on local.
```

File: test/images/agnhost/README.md [hunk 2]
```diff
@@ -164,9 +164,14 @@ Usage:
 
 ### fake-gitserver
 
-Fakes a git server. When doing `git clone localhost:8000`, you will clone an empty git
-repo named `8000` on local. You can also use `git clone localhost:8000 my-repo-name` to
-rename that repo.
+Fakes a git server. When doing `git clone http://localhost:8000`, you will clone an empty git
+repo named `8000` on local. You can also use `git clone http://localhost:8000 my-repo-name` to
```
Comment (spiffxp) [inline test/images/agnhost/README.md]:
```suggestion
repo named `localhost` on local. You can also use `git clone http://localhost:8000 my-repo-name` to
```

**Review by spiffxp — CHANGES_REQUESTED** [review]
When I try cloning with no name, I end up cloning into `localhost`.  I don't think this ever cloned into `8000` since it was added via https://github.com/kubernetes/kubernetes/pull/19857
```
spiffxp@spiffxp-macbookpro:agnhost (master %)$ go run agnhost.go fake-gitserver&
[1] 33375
spiffxp@spiffxp-macbookpro:agnhost (master %)$ git clone http://localhost:8000
Cloning into 'localhost'...
warning: You appear to have cloned an empty repository.
spiffxp@spiffxp-macbookpro:agnhost (master %)$ git version
git version 2.18.0
```

File: test/images/agnhost/fakegitserver/gitserver.go (hunk 1 above)
Comment (chendave) [reply test/images/agnhost/fakegitserver/gitserver.go]: Good catch!

**Review by spiffxp — APPROVED** [review]

**Review by claudiubelu — APPROVED** [review]

**Pull request merged**